sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_HealthcareAsync_AutomaticPolling.cs

```diff
@@ -38,30 +38,31 @@ public async Task HealthcareAsync_AutomaticPolling()
 
             await healthOperation.WaitForCompletionAsync(pollingInterval);
 
-            AnalyzeHealthcareEntitiesResultCollection results = healthOperation.Value;
-
-            Console.WriteLine($"Results of Azure Text Analytics \"Healthcare Async\" Model, version: \"{results.ModelVersion}\"");
-            Console.WriteLine("");
-
-            foreach (AnalyzeHealthcareEntitiesResult result in results)
+            await foreach (AnalyzeHealthcareEntitiesResultCollection documentsInPage in healthOperation.Value)
```

Comment:
Updating the samples made me realize that some are not needed anymore, or the input doesn't reflect the purpose, so they definitely need some love.
To avoid making this PR bigger, that work will come in a future PR

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/17810